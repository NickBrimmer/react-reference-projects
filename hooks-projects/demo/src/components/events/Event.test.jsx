import React from 'react';
import { shallow } from 'enzyme';
import Event from './Event';

describe('Event component', () => {
  it('renders Event', () => {
    const wrapper = shallow(<Event year="1455" text="Stuff happened" />);
    expect(wrapper).toMatchSnapshot();
  });
});
  
