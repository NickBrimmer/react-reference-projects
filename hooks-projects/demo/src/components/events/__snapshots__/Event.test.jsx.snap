// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event component renders Event 1`] = `
<p>
  1455
   - 
  Stuff happened
</p>
`;
